exporter/signalfxexporter/exporter.go

```diff
@@ -99,9 +100,19 @@ func newSignalFxExporter(
 			ingestURL: options.ingestURL,
 			headers:   headers,
 			client: &http.Client{
-				// TODO: What other settings of http.Client to expose via config?
-				//  Or what others change from default values?
 				Timeout: config.Timeout,
+				Transport: &http.Transport{
+					Proxy: http.ProxyFromEnvironment,
+					DialContext: (&net.Dialer{
+						Timeout:   30 * time.Second,
+						KeepAlive: 30 * time.Second,
+					}).DialContext,
+					MaxIdleConns:        config.MaxConnections,
+					MaxIdleConnsPerHost: config.MaxConnections,
+					IdleConnTimeout:     30 * time.Second,
+					TLSHandshakeTimeout: 10 * time.Second,
+					ForceAttemptHTTP2:   true,
+				},
```

Comment:
Instead of doing this and risk diverging when Go http package changes the defaults, I think something like this is less code and is likely safer long term:
```go
transport := http.DefaultTransport.(*http.Transport).Clone()
// Modify what needs to be modified: 
transport.MaxIdleConns = ...
```

Reply:
Thanks, @tigrannajaryan  for drawing attention, We have made changes as per your suggestion.